chore: move to v1

henryiii opened the pull request:
Updating to recipe v1. Started with `pixi exec conda-recipe-manager convert recipe/meta.yaml`.

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->
Closes: https://github.com/conda-forge/numba-feedstock/pull/119

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

MementoRC committed:
chore: rebase on 0.66.0

MementoRC committed:
chore: tweak CI runners

MementoRC committed:
chore: tweak tests

MementoRC committed:
fix: runtest argument syntax

MementoRC committed:
ref: unify non-unix to sh with new rattler

MementoRC committed:
fix: var typo

MementoRC committed:
fix: var typo

MementoRC committed:
fix: revert osx-64 to partial test. added partial testing (but for build_number==0)

MementoRC committed:
fix: odd loss of osx/non-unix

MementoRC committed:
fix: Don't work when alma9->alma10! catchsegv

henryiii committed:
fix: kill lingering mspdbsrv.exe so Windows test cleanup succeeds

MSVC's mspdbsrv.exe outlives the test script and holds a handle in the
test dir, making rattler-build's cleanup fail with 'Access is denied
(os error 5)' after all tests pass (prefix-dev/rattler-build#2657).

Assisted-by: ClaudeCode:claude-opus-4.8

MementoRC committed:
fix: More aggressive non-unix coercion

MementoRC committed:
fix: cd sufficient

MementoRC committed:
fix: non-unix annoyance

MementoRC committed:
fix: non-unix annoyance

MementoRC committed:
fix: non-unix annoyance: python

MementoRC committed:
fix: capitulate non-unix

MementoRC committed:
fix: giving up, non-unix fails seem random

MementoRC committed:
dbg: 25% for triangulation of culprit

MementoRC committed:
dbg: Seems random, last try

MementoRC committed:
dbg: probe to 45s for rattler delete

MementoRC committed:
dbg: what if we add a simple test

MementoRC committed:
dbg: attemp log size reduction

MementoRC committed:
dbg: attemp log size reduction -> TEMP?

MementoRC committed:
fix: --random to dodge rattler-build #2657 cleanup race

henryiii committed:
chore: re-render with conda-forge-pinning 2026.07.29.10.37.28

tbb2023 migration is now in the global pinning.

Assisted-by: ClaudeCode:claude-fable-5
